test/src/main/java/org/apache/accumulo/test/ComprehensiveTableOperationsIT.java

```diff
@@ -0,0 +1,957 @@
+      // removeProperty
+      ops.removeProperty(tableName, propKey);
+      // getProperties, getTableProperties, and getConfiguration
+      assertFalse(propFound(tableName, propKey, newPropVal));
+      assertNull(ops.getTableProperties(tableName).get(propKey));
+      assertNull(ops.getConfiguration(tableName).get(propKey));
+    }
+  }
+
+  @Test
+  public void test_setLocalityGroups_getLocalityGroups() throws Exception {
+    // setLocalityGroups, getLocalityGroups for user tables is tested in LocalityGroupIT. Ensure
+    // test exists
+    assertDoesNotThrow(() -> Class.forName(LocalityGroupIT.class.getName()));
+    // setLocalityGroups, getLocalityGroups for system tables not tested for. Test basic
+    // functionality here
+    for (var sysTable : AccumuloTable.values()) {
+      LocalityGroupIT.createAndSetLocalityGroups(client, sysTable.tableName());
+      LocalityGroupIT.verifyLocalityGroupSet(client, sysTable.tableName());
+    }
+  }
+
+  @Test
+  public void test_importDirectory() throws Exception {
+    // importDirectory for user tables is tested in BulkNewIT. Ensure test exists
+    assertDoesNotThrow(() -> Class.forName(BulkNewIT.class.getName()));
+    // importDirectory for system tables not tested for. Test basic functionality here
+    var rootPath = getCluster().getTemporaryPath().toString();
+    var dir = rootPath + "/" + getUniqueNames(1)[0];
+    BulkNewIT.writeData(dir + "/f1.", getCluster().getServerContext().getConfiguration(), 0, 5, 0);
+
+    for (var sysTable : AccumuloTable.values()) {
+      assertThrows(Exception.class, () -> ops.importDirectory(dir).to(sysTable.tableName()).load());
+    }
+  }
+
+  @Test
+  public void test_offline_online_isOnline() throws Exception {
+    // offline,online,isOnline for user tables is tested in ComprehensiveBaseIT. Ensure test exists
+    assertDoesNotThrow(() -> Class.forName(ComprehensiveBaseIT.class.getName()));
+    // offline,online,isOnline not tested for system tables. Test basic functionality here
+    for (var sysTable : AccumuloTable.values()) {
+      switch (sysTable) {
+        case ROOT:
+        case METADATA:
+          assertTrue(client.tableOperations().isOnline(sysTable.tableName()));
+          assertThrows(AccumuloException.class,
+              () -> client.tableOperations().offline(sysTable.tableName(), true));
+          assertTrue(client.tableOperations().isOnline(sysTable.tableName()));
+          client.tableOperations().online(sysTable.tableName(), true);
+          break;
+        case SCAN_REF:
+        case FATE:
+          assertTrue(client.tableOperations().isOnline(sysTable.tableName()));
+          client.tableOperations().offline(sysTable.tableName(), true);
+          assertFalse(client.tableOperations().isOnline(sysTable.tableName()));
+          client.tableOperations().online(sysTable.tableName(), true);
+          assertTrue(client.tableOperations().isOnline(sysTable.tableName()));
+          break;
```

Comment:
This will be changed once #5433 is completed and merged in: should not be able to offline any system table

Reply:
#5433 has been merged in. This test has been updated (in 4eb99ebd482139c834dbad28948a1842ab88b7d5) to expect an exception when offlining any system table